theano/sandbox/gpuarray/tests/test_type.py

```diff
@@ -1,12 +1,22 @@
+import os
+
+from nose.tools import assert_raises
 import numpy
 
 import theano
+from theano.compat import PY3
+from theano import config
 from theano.compile import DeepCopyOp
+from theano.misc.pkl_utils import CompatUnpickler
 
 from .config import test_ctx_name
 from .test_basic_ops import rand_gpuarray
 
 from ..type import GpuArrayType
+from .. import pygpu_activated
+
+if pygpu_activated:
```

Comment:
Since you import `.config`, you can just import pygpu directly.

The `.config` import will raise an exception if pygpu is not available.